src/DataFramesMeta.jl

```diff
@@ -44,6 +44,86 @@ replace_syms!(e::Expr, membernames) =
         mapexpr(x -> replace_syms!(x, membernames), e)
     end
 
+"""
+    @col(kw)
+
+`@col` transforms an expression of the form `z = :x + :y` into it's equivalent in
+DataFrames's "mini-language".
```

Comment:
"mini-language" is probably too strong... :-)
```suggestion
DataFrames's `source => fun => destination` syntax.
```